src/cgnstools/cgnsplot/CMakeLists.txt

```
@@ -46,7 +46,7 @@ if (WIN32)
   endif (HTML_HELP_INCLUDE_PATH AND HTML_HELP_LIBRARY)
 else (WIN32)
   include_directories(${X11_X11_INCLUDE_PATH})
-  link_libraries(${X11_X11_LIB} ${X11_Xmu_LIB} m)
+  link_libraries(${X11_X11_LIB} ${X11_Xmu_LIB} m uuid)
```

Not sure if I should check for uuid and add a dependency.

---

Instead of adding uuid, adding ${X11_SM_LIB} should be better. uuid is supposed to be set by X11 in this variable

```suggestion
  link_libraries(${X11_X11_LIB} ${X11_Xmu_LIB} ${X11_SM_LIB} m)
```

---

Thanks, fixed.